packages/react/src/components/Forms/Switch/tokens.ts

```diff
@@ -0,0 +1,11 @@
+import { DefaultTokens } from "@openfun/cunningham-tokens";
+
+export const tokens = (defaults: DefaultTokens) => ({
+  "accent-color": defaults.theme.colors["success-600"],
+  "rail-background-color": defaults.theme.colors["greyscale-500"],
+  "rail-background-color--disabled": defaults.theme.colors["greyscale-400"],
+  "rail-border-radius": "34px",
```

Comment:
A little trick if you do not want manage tricky values.
```suggestion
  "rail-border-radius": "50vw",
```